python/ray/serve/_private/utils.py

```diff
@@ -473,3 +473,27 @@ def noop_decorator(func):
             return func
 
         return noop_decorator
+
+
+def snake_to_camel_case(snake_str: str) -> str:
+    """Convert a snake case string to camel case."""
+
+    words = snake_str.lstrip("_").strip("_").split("_")
```

Comment:
```suggestion
    words = snake_str.strip("_").split("_")
```
strip does lstrip's job already

Reply:
Thanks for letting me know. I removed the `lstrip` call.